nuxeo-runtime/nuxeo-runtime-deploy/src/main/java/org/nuxeo/runtime/deployment/NuxeoStarter.java

```diff
@@ -96,7 +96,11 @@ public void contextInitialized(ServletContextEvent event) {
             Double duration = (finishedTime - startTime) / 1000.0;
             log.info(String.format("Nuxeo framework started in %.1f sec.", duration));
         } catch (IOException | BundleException e) {
+            log.error("Exception during startup", e);
```

Comment:
Will it be redundant to log the stack trace meanwhile it is re-thrown?

Reply:
Maybe but I'd rather be safe, just in case these exceptions are swallowed by a caller. Which is why I'm adding this in the first place (for the `Error` case)...